spec/orbital/remote/request_spec/tokenized_creditcard_payload_spec.rb

```diff
@@ -0,0 +1,138 @@
+require 'spec_helper'
+
+ActiveMerchant::Billing::Base.mode = :test
+
+describe 'Payment request for network tokenized card' do
```

Comment:
Since these are all fast (stubbed Orbital) tests, they should be in `spec/orbital`, not `spec/orbital/remote`.